libkbfs/folder_branch_ops.go

```diff
@@ -4393,7 +4398,7 @@ func (fbo *folderBranchOps) notifyOneOpLocked(ctx context.Context,
 	// We need to get unlinkPath before calling UpdatePointers so that
 	// nodeCache.Unlink can properly update cachedPath.
 	unlinkPath, unlinkDe, toUnlink, err :=
-		fbo.getUnlinkPathBeforeUpdatingPointers(ctx, lState, md.ReadOnly(), op)
+		fbo.getUnlinkPathBeforeUpdatingPointers(ctx, lState, md, op)
```

Comment:
why remove the `.ReadOnly()`?

Reply:
Because `md` here is already a `ReadOnlyMetadata` -- it was an extraneous call, I guess leftover from some old core.  Nothing to do with the fix, just something I noticed while debugging.